src/views/TabView/TabBarBottom.js

```diff
@@ -153,10 +153,13 @@ export default class TabBarBottom extends PureComponent<
             outputRange: (outputRange: Array<string>),
           });
           const justifyContent = this.props.showIcon ? 'flex-end' : 'center';
+          const accessibilityLabel = route.accessibilityLabel || route.title;
           return (
             <TouchableWithoutFeedback
               key={route.key}
               onPress={() => jumpToIndex(index)}
+              accessible={true}
+              accessibilityLabel={accessibilityLabel}
```

Comment:
Is there ever a situation where the route title would be undefined? This would negate the `accessible={true}` if there is an `undefined` label.

Reply:
Perhaps the `accessible` boolean should only be `true` when a title OR label is detected.